### ISSUE TITLE
Posts containing specific URLs are not reflected in bluesky

### ISSUE THREAD

**Issue by chaperon1234** [snarfed/bridgy-fed#1591]
https://github.com/snarfed/bridgy-fed/issues/1411
I have replied to the CLOSED issue, but please let me make another article as a new issue.

[
My bluesky account is
https://bsky.app/profile/chaperon.fedibird.com.ap.brid.gy
but these toot is not reflected in blue sky.

https://fedibird.com/@chaperon/113585855054825034
https://fedibird.com/@chaperon/113581312460836009

There are several other posts that have not been reflected in blueSky since the day the link preview feature of this issue was implemented.
It appears to be mainly Twitter and Bluesky links, but there are a few other posts that are not being reflected.
Do you know the cause?](https://github.com/snarfed/bridgy-fed/issues/1411#issuecomment-2513369779)

**Comment by snarfed** [comment]
Thanks for filing! Looks like a straightforward bug with our domain blocklist, which includes x.com, bsky.app, etc. Will fix.

**Comment by snarfed** [comment]
Fix ^ should be deployed in ~10m. Thanks again for reporting!

**Comment by chaperon1234** [comment]
I have verified a few things and I don't think this issue has been resolved yet.
It seems that the link preview does and does not open for some URLs.

Link opens: (OK case)
https://bsky.app/profile/chaperon.fedibird.com.ap.brid.gy/post/3lcen2o2fmqb2


Link does not open: (NG case)
https://bsky.app/profile/chaperon.fedibird.com.ap.brid.gy/post/3lchk7mg7apq2
https://bsky.app/profile/chaperon.fedibird.com.ap.brid.gy/post/3lcj7maovwda2
https://bsky.app/profile/chaperon.fedibird.com.ap.brid.gy/post/3lcf44xx4ueh2

Of the failing cases, I thought that the last URL might be because the account owner has selected a setting not to be displayed from logged out users. But the other two should be displayed? I think.
